src/Extension/AccountReset/SecurityExtension.php

```diff
@@ -102,33 +100,23 @@ public function resetaccount(HTTPRequest $request)
 
     public function ResetAccountForm(): Form
     {
+        $field = ConfirmedPasswordField::create(
+            'Password',
+            _t(Member::class . '.NEWPASSWORD', 'New Password'),
+            titleConfirmField: _t(Member::class . '.CONFIRMNEWPASSWORD', 'Confirm New Password')
```

Comment:
You can't use named arguments with `create()`.

Reply:
Updated